src/graph.js

```diff
@@ -40,37 +40,111 @@ class Graph {
   // Optionally accepts an array of other GraphNodes for the new vertex to be connected to
   // Returns the newly-added vertex
   addVertex(value, edges = []) {
-
+    const newNode = new GraphNode({ value, edges });
+    edges.forEach((edge) => {
+      edge.pushToEdges(newNode);
+    });
+    this.vertices.push(newNode);
+    if (this.vertices.length === 2) {
+      this.vertices[0].pushToEdges(this.vertices[1]);
+      this.vertices[1].pushToEdges(this.vertices[0]);
+    }
+    return newNode;
   }
   // Checks all the vertices of the graph for the target value
   // Returns true or false
   contains(value) {
-
+    let findTarget = false;
+    this.vertices.forEach((vert) => {
+      if (vert.value === value) {
+        findTarget = true;
```

Comment:
You can `return` right after you've found the target value. That way your function won't do any extra work after you've already found the target you're searching for.